### PR TITLE
feature/AP5-27

### PR DESCRIPTION
**Pull request by mcraepv**

### CODE REVIEW

File: rigex-front-end/src/components/WellsDropdown/WellsDropdown.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        case 'edit':
+          setIsOpen(!isOpen);
+          break;
+        case 'move':
+          setIsOpen(!isOpen);
+          break;
+        default:
+          setIsOpen(!isOpen);
+          break;
+      }
```
Comment (popesicle) [inline rigex-front-end/src/components/WellsDropdown/WellsDropdown.js:38]: Try using a fall-through for the switch cases that do the same thing. 

`case 'edit':
 case 'move':
 default:
`
I can't remember if default can actually go in there, but give it a try and see if it works like you're wanting.

Comment (popesicle) [reply rigex-front-end/src/components/WellsDropdown/WellsDropdown.js:38]: Obviously formatted correctly lol

Comment (mcraepv) [reply rigex-front-end/src/components/WellsDropdown/WellsDropdown.js:38]: I thought about that, the problem was that every case-specific operation would run from the top down, not just the setIsOpen. I guess I could do another name==='case' check but that seemed to defeat the purpose of the switch lol

**Review by popesicle — COMMENTED** [review]
Looks good, left a comment.

File: rigex-front-end/src/components/WellsDropdown/WellsDropdown.js (hunk 1 above)
Comment (popesicle) [reply rigex-front-end/src/components/WellsDropdown/WellsDropdown.js:38]: Alright, that makes sense.

**Review by popesicle — APPROVED** [review]

**Review by BarretteBanner — APPROVED** [review]

**Pull request merged**